src/index.js

```diff
@@ -200,6 +204,72 @@ class IpfsRepo {
   exists (callback) {
     this.version.exists(callback)
   }
+
+  /**
+   * Get repo status.
+   *
+   * @param {function(Error, Object)} callback
+   * @return {void}
+   */
+  stat (callback) {
+    waterfall([
```

Comment:
Here you're using waterfall to pass the accumulator, but since, by what I gather, each step is creating it's own entry, perhaps you could use [`async/parallel`](https://caolan.github.io/async/docs.html#parallel) with the object arguments, collecting them all automatically in the final callback.

Reply:
Multiple steps use the `repoSize`. If it was parallel I think I could end up messed up.

Reply:
So perhaps you could calculate that first, and then proceed to make the rest parallel?

Reply:
Yeah!